app/models/job_execution.rb

```diff
@@ -106,8 +106,9 @@ def stage
   end
 
   def error!(exception)
-    puts_if_present report_error(exception)
-    puts_if_present "JobExecution failed: #{exception.message}"
+    puts_if_present exception.message, "JobExecution failed: #{exception.message}"
+    error_url = report_error(exception)
+    puts_if_present error_url, "Error URL: #{error_url}"
```

Comment:
instead of adding an argument `puts_if_present "Error URL: #{error_url}" if error_url`

Reply:
Hm.. but then what is the point of `puts_if_present`?

Reply:
ah yes, just puts then